static/templates/search_list_item.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="search_list_item">
+    {{{ description_html }}}
+    {{#if is_person}}
```

Comment:
Why did you add this condition if you always set 'is_person' to true in `search_suggestion.js`?

Reply:
`is_person` is not true (actually it's simply not defined) for stream suggestions, etc, so this condition is necessary